core/src/lib.rs

```diff
@@ -291,12 +297,60 @@ impl Schema {
     pub fn identifies_attribute(&self, x: &NamespacedKeyword) -> bool {
         self.get_entid(x).map(|e| self.is_attribute(e)).unwrap_or(false)
     }
+
+    /// Returns an edn::Value::Vector representation of the schema map.
+    pub fn to_edn_value(&self) -> edn::Value {
+        let mut all_values = Vec::new();
+        for (entid, attribute) in &self.schema_map {
+            let mut attribute_map: BTreeMap<edn::Value, edn::Value> = BTreeMap::default();
+            let some_ident = self.get_ident(entid.clone());
+            if some_ident.is_some() {
+                let ident = some_ident.unwrap();
+                attribute_map.insert(values::DB_IDENT.clone(), edn::Value::NamespacedKeyword(ident.clone()));
+            } else {
+                attribute_map.insert(values::DB_IDENT.clone(), edn::Value::Keyword(Keyword::new(entid.to_string())));
```

Comment:
This isn't right: you'll be making something like

```edn
  :db/ident :1234
```

Just don't do it at all. This whole block can become:

```rust
if let Some(ident) = self.get_ident(entid) {
    attribute_map.insert(values::DB_IDENT.clone(), edn::Value::NamespacedKeyword(ident.clone()));
}
```